app/models/feed_token.rb

```diff
@@ -3,6 +3,7 @@ class FeedToken < ApplicationRecord
 
   validates :token, presence: true, uniqueness: {scope: :feed_id}
   validates_format_of :token, with: /\A[0-9a-zA-Z_.-]+\z/
+  validates :label, presence: true
```

Comment:
added this validation since it seems to be a form validation in Publish. (can't save without a label)